lib/group/group-router-spec.js

```diff
@@ -0,0 +1,218 @@
+var sinon = require('sinon');
+var assert = require('assert');
```

Comment:
We could take the opportunity to include `chai` here as well. `require('chai').assert` is API compatible.